central/complianceoperator/v2/checkresults/datastore/datastore_impl_test.go

```diff
@@ -319,58 +319,6 @@ func (s *complianceCheckResultDataStoreTestSuite) TestDeleteResult() {
 	s.Require().Nil(retrieveRec1)
 }
 
-func (s *complianceCheckResultDataStoreTestSuite) TestSearchCheckResults() {
```

Comment:
These were duplicative as the cases were covered in the more robust tests for SAC.